azext_iot/iothub/providers/device_identity.py

```diff
@@ -0,0 +1,426 @@
+                        destination=device_cert_output_directory,
+                        file_name=f"{device_id}.hub-auth-key.pem",
+                        overwrite=True
+                    )
+                else:
+                    device_keys = device_result.authentication.symmetric_key
+                    device_pk = device_keys.primary_key if device_keys else None
+
+                # edge device config
+                create_edge_device_config(
+                    device_id=device_id,
+                    hub_hostname=self.target["entity"],
+                    auth_method=config.auth_method,
+                    default_edge_agent=config.default_edge_agent,
+                    device_config=device_config_dict[device_id],
+                    device_config_path=config.template_config_path,
+                    device_pk=device_pk,
+                    output_path=device_cert_output_directory,
+                )
+                # root cert
+                write_content_to_file(
+                    content=config.root_cert["certificate"],
+                    destination=device_cert_output_directory,
+                    file_name=root_cert_name,
+                )
+                # full-chain cert
+                make_cert_chain(
+                    certs=[
+                        signed_device_cert["certificate"],
+                        config.root_cert["certificate"],
+                    ],
+                    output_dir=device_cert_output_directory,
+                    output_file=f"{device_id}.full-chain.cert.pem",
+                )
+                # write install script
+                write_content_to_file(
+                    content=create_edge_device_config_script(
+                        device_id=device_id,
+                        hub_auth=hub_cert_auth,
+                        hostname=device.hostname,
+                        has_parent=(device.parent_id is not None),
+                        parent_hostname=device.parent_hostname,
+                    ),
+                    destination=device_cert_output_directory,
+                    file_name="install.sh",
+                    overwrite=True,
+                )
+                # write device readme
+                write_content_to_file(
+                    content=DEVICE_README,
+                    destination=device_cert_output_directory,
+                    file_name="README.md",
+                    overwrite=True,
+                )
+                # create archive
+                tar_directory(
+                    target_directory=device_cert_output_directory,
+                    tarfile_path=bundle_output_directory,
+                    tarfile_name=device_id,
+                    overwrite=True,
+                )
+                # delete uncompressed files
+                rmtree(device_cert_output_directory)
+
+        # Give device registry a chance to catch up
+        sleep(1)
```

Comment:
this may not be enough - we should have retries with sleeps or have a better error 

<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAACJ0AAAC6CAYAAAAOTK4dAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAADIUSURBVHhe7d27ji1LmhDgrtZ2NhY2T9HSAeloDMYACSSeAs0TjIeDhYM3FgZCI54CCSQwBmN0nJb6KbCxMGej6K5/5u/ouOVt5Vqrvk9K9cqM+C+Ra9U+NZUxVR8f377/+BUAAAAAAAAAAGzw68//BQAAAAAAAACAZTadAAAAAAAAAACwmU0nAAAAAAAAAABsZtMJAAAAAAAAAACb2XQCAAAAAAAAAMBmNp0AAAAAAAAAALCZTScAAAAAAAAAAGxm0wkAAAAAAAAAAJvZdAIAAAAAAAAAwGY2nQAAAAAAAAAAsJlNJwAAAAAAAAAAbHb5ppNffvu7z1fP4ax+/vl/+1d/f2w1ilvJOxt/Nf/uf/6/z1fXK7UeWe8ZvON6n/V9jL5m/c3G392zrP/nv/nbz1ev6+/++q9+f1zl6vwzV9S/a0131M0143W+tsWZXy/P9r1hKH09a28td/d6dv1Xu/+ca/T+x9jo8zEbP+Jo3qv64nmMPn8xNvoczMafzdm93r3+rfXv7JW5s9+frZ+Ps22tf2avK7lG/cXYSp6wZS48m7M/v1u/fs62tf6dvQIAz2G66eT//ut/8UfHFuWbjZ9/+s3n2XMo/Rz9Jqhs+Pjf/+Z//P1x5gaQyHnEaj9nb1zZ+s1o8ZUfvD+C+/unrtzw8B//5T/6/fHMrlz/KykP0H/58z/7PHucvQ/84ahf/8Vf/v7Yq3y9HN14kr9POPq92DN6xzXxPL7y56v8329H/2/KV79/V/d/Vv5ennI9H7XZ+Cvz+eXZnfX58jmldsa/f6ve9fP37l9X/v3hmR39XJX4fADAVzDcdFI2mfzj//6//uhY3XhS/mP6qP/jYqvS197/2MeGk2zrxpOjG0uOxhdnbzjJtt7bZ39I/+re8f6+wuaOkVfv/6i713/XhhOew9HNF1/ZkY0n8X1hfG/4jN8j5v54PPf/azv6/vv83G/2fwPO3qNXfg+P9v7Kaz/D3ev/6vf/DHt/vvYKfD7HnqG/d/78Meffn+s8w9f3q8o//4jDv1UAfAUfH9++//h8/Udiw0n8b+9aT/zHtaUea51nrbkxp1Vjlr9oXVvR2nRS5Ov1nHosjDaOjOqEejyPZa155VqvxhHxvhSz+xu/bSEePpfz1oPo1vX6NzXkHEU5r/MX9bXRnDAaC3lOkefVY8WoRrGlh1b+IubN6o9yh5hT5xyd12NZXac1p2U1rjWvvhZWa2ezPmK8dz3sqV2M1jerWWv10OuzlWtWL4+PxrIj9Vf0Np20fgNJa3NCPS/P6Y31frvJ1vzFap89R/KvrGOWf2a2vl7+uF7XK9fLtdW+ZvNW89RyfzlHxOfxoneetWrP5tV5a3V8PW/vpq34vmvv918hvs9ofV9X21Jn5fuXukbMa9Uutq6zlz8bzWn1MRrf2l+xpX6dv4zX8+M8x/b6WsmfbR2fGcWXsXIec1q5V+q34uu4UMev5N+r5C75Wv2tihwhn+feR7nrHGEUn8eyUZ1a1I1crRr5Wu884os6R0+OyVo9ZKvjq/lnSp4SE/+bza6txhxR8tW25K/7qfsPo5y9NY3i81g2qnOVupfcw2isqNeez3Nsb10r+bNenpaV+sXeGiWuzJ3VifF6rI4fxYY8ZxRfx4VWjZFcY2vsWeq1jNZZ91j3nc9zbG9tK/mzXp6WlfrFWTWKHFvGynnMqfNu6W82no3m1nKPW+KKHBtaOWK8HqvjR7EhzxnF13GhVaOnzl+04lv1izp+FBvynFF8HRdaNUZyja2xZ6nXMlpn3WPddz7Psb21reTPenlaVuoXe2uUuDJ3VifG67E6fhQb8pxRfB0XWjW2iJpnqfOdnR8A9pj+eZ3YZLKy0SQc+Y9c/Ie9xEeOuBYifznqsVV7YvMmjfI6ji1K/JGNHqP4PBavW3Nb11aVezY6svr8LPHguTxs7j2MLuejB9VFzhPq3HmslsdzjajdGitmNerxIq7NYleUHK3cRStvfr0qasTrrF5DUc/pyTE9vfx1bH2+qs6/JcfemtloffE6jHqtz0Mvf601NqpXtK7VjtRfsfrAvPegPz+wr+fUYzFezM7DKH/RGt/iSP7eWH49yz8zql9szZ/HejmzOn89f2v9ljL/SHzRi231t0Urvq5x9LedHFHie9//xfVyxPkWObYl6uV5rWvxOs5XjfKHct7L3xrLc1byz8zqF3vzt3K2xLx67qx+3XvM2WoUn2vUa5/1V/Tic0y8jvOwkv+oXn9nyH3vMYrPY/G6N3fk6PpzfJyvyDHxOs5D5MpjOX+unecU+bw1vmpLzGztq/dmVb3+Ys8ae3LePUbxeSxe9+ZeqXUPQ7xfeWzLe9jK2RLz6rmPqD9a/4ocH+dZHm/1PhqP8xgvWnNa8TkmXsf5FntizpTXV/eycn9GWjlbYl499xH1R+tfNYrP+eveezFbHO3/zPpxno3WX4zG4zzGi9acVnyOiddxvkXOH+dZr34Yjcd5jBetOa34HBOv43yLPTFnyuure1m5PyOtnC0xr577iPqj9a/I8XGe5fFW76PxOI/xojWnFZ9j4nWcAwBj000nV6n/Q5/l/5C3/qO+8h/6Uf6jYqPJaPPG1s0oX01+sNzSe5A8iwuz8b31wyj/aGy17mr/PaM8s9xnOLq+vV49/8ysflwv8/b0uhozG488W51V/6jew/r8QH6kfki/apZ/tX7P3flnjva39Xrt6vsTroq/+v4flX9QU75H2/N92sr3f1fK9a/oZZS/3K+jNY/0v1J/Jf/e78/Pqr/X1vots/6O9jzLf9QZOfe+/8/gld6fK/Kfobz/o6+l2fjdXvnze8TKe7Ly+dt7/86qv9esfhlvHdmsv1nPW8b35H9l5V4fvT9F/Z6tOqv+XrP6Zbx1hPJ6S/9nW6l/tVy/1cvR+3M0/0h+T/ORHa2/ZXxP/ldW7vXR+1PU79mqs+rvNatfxltHNutv1vOW8T35z1TWfkW9+p4CwN2mm07iN5zEbzw5Q/6PbOs/uvGNSBxbzfKfYbThJI+VzSdHfrPIsyn3cnRk9fmK+mH13ofXoTyc3vKAOs8ttffUj7gjvfdyPKq/GDv6cL93/0c95LHW+NVeof5oPN/vve/fKP9Irrcn/lWUh/NxZPlhfWt8VS//Wd49f1y/q/6re8T64vuDrd/jrXxveNX3fY+Q1zda415358/vSxk/+30a1S+1emNXaK0t12/1MBpbkeP35DgaP3P1+3/U1eufOVp/FH/G5z/H78lRYkofvfd9Nj6TeyvH2XJf0esrmd2f2fjMLP7q+zerf6WyltaxxZbeW7lz/EqOZzPrfzY+M4vP97SMb33/Zmb1rxSfx/pg3Zb3rnVvc/xKji3q9zWOLbb01sqd41dyPJtZ/7PxmVl8vqdlfOv7NzOrf6X4PNbHFlt6b+XO8Ss5rrK39qz/vOYyvvX+AsAVuptOYpNJ/pM6rWtXiG9E8sHzy98ArbxnvQfi5UF1GbvygXXUbtUoY3EUW/qIuTl+j9xDnau+1usvx4Qt/c3Gj4j6+Qi964/yCvXrOXleeY/jfMtnN6tzR74VdczeHp5Z2VxSH6G+tufBfc6Rc53laP68IaEVW+duzTni6vwzd9e/2qPWt/V7u/ghyuj7wlf/QUteXxxblPXH0Yqtc7fmHHF1/plZ/Xy93KNHy/XjCPGe1de3yPF78hyJfQd3r/9o/Tq+zpGv7/n85/hybDWL2ZMzy721csW/jfG19tXM7s9sfKaO35PjiDvr589WPlaVuUf7zvFH8txl1vtsfKaO35PjiDvr15/LOFhT7tXR9y3HH8nTUr+vcawqc4/2leOP5LnLrPfZ+EwdvyfHEXfWrz+Xcawqc4/2neOP5Dki1ryn9t29A8Aew990EptM8nH1hpNiyzchPJ/ZN0JbNkkUret1jl7s1aKPrfVnD+KPPKg/Elusxt/Z40yd/+zPx9X9z/Tq5+ux5j29Ptv67vr67rnyN0AUq/n39nE0/2wzwln3p5dnlL/uac+GiVn/Z6+v1eMZNfbmOGt9V7rr+8Rc94oeZjln9Wc/kDra86z+0fwzs/x3159Zjd9b5+r1s2blfShzzv7B8TO//2Wtub96/bPxM5R8cbBd/f7UWtfONMv/yPp1rfzZykdPie+N713H1et/dqP3p7j6/szyP7J+Xav+XMaRXd3fzKj/Ryv16/sT9vZ25Zrq9zWOnldb3yvI62/di6vvzyz/I+vXterPZRw9Jb43vncdV69/JuqP1g0A7+bj49v3H5+vTzX6ZiH05rS+KYh5dcyozt6xnvhTOeV/a60/odP60zqt2CLmHR0Ped5KD/WcK5UHyr2HyPGwuYzn19nogXQvJtTjvfOslT+ra+2Zs5pjNXd9LRvlaI0VOd9K/Kh+Mcoxsqe/Vt48Z6VurVdj1t9K/yt6eYqcq7fO3vXQyl/mxfWI6Z1nvbrZaE5rrFd/1c9/87e/+uXP/+zz7B/UmwB6mwJaD+XLnN71Wp63uuGg7qnWytOzN/9q7VH+mZUas/x5fBZXzPqfjRf1nJaVuNGcGOudF634oswZjWWjHore18/M6Hu7VXWO+vvB2pb8rfhiVKOVP885ur6iV79cL+cxPostVuaMjOoXq/nruKIVW4zyr+SIObPYFaMcZWyWc9RfsdJjnlOPz/IfUa+vPt+iFdvqvYh5R8dDnrel/7rn+ryI3OV6a362pXaW8/TqZzFntf4o/8yofjHrYbXHPWa9bVFyzXoPMe/oeMjz9va/V127nOceWmto9VjHFa3YYpR/JUerfsvW+uV6Od+bvxU3mrNSq44vIqaO7+XLOWb1aqP6j1D33lpzrXcP6uut2GKUfyVHq37L1vrlejlfzV+M+q9ztc5bYs5svMhzyvW6xkyrxmp8HduKG81Z6XXUXx3fy5dzzOplde1Z7mKln6yOLyKmju/lyzlm9Wqj+o9Q995ac613D+rrrdhilH8lR6t+y9b65Xo535u/FTeas1Krji8ipo7v5cs5ZvWyVu1iS45Vvd4B4A63bjq5057+ymaNR27OeEd7HxI/i2fv/9XvLzzC3ofmX91ok0O9cYH3dfTrxw9ErtG6r+41gH8fuZfPGjyvd//69O8Pz8znEwDe0/DP6xxRvnEo30A8o73f2JQNJ/VvCen95hHaymYIGyKu4/7CXHlgXh6cA9vYsAUAAAAAANQu+00n4dl2rp7Rz+hP1/De/CYReB/v9AC996dTwlm/iWT2p1d6HtXfXd59fYUNJ8+v3uzt/3MM4A/8+8hdnu3nYVvUXzc1X0dc6RGfv1f++lzh3x+emc8nALynyzedAAAAAAAAAADwfi778zoAAAAAAAAAALwvm04AAAAAAAAAANjMphMAAAAAAAAAADZ7+KaTn//mbz9f9ZU5K/P4g6336qvf31def/T+TP3f3ctXqh/v/d1r/grc63O4f+vu/sw9Q/13M1vTljW/4v15x/e0KOuKAwAAAACAkzed5B/C1gfwB0e+Hn758z/7/cHX9A7v/5X/Pcj/zTlSp8TGvfb1xhZHPnd3f95e4fM+u79H7v/Z4t+Rntn4q+utr1yPo9a6FnJcHHcodeNrZev7d7Tnu9YMAAAAADBz6qaT+oew9fmqPTGs++r31+cL3ld8fZfDAzqA51Fv2Njyb3TE1K8BAAAAALjfx8e37z8+X58qfrBci+vxg+Z6Tv4BdO8HyvUPqbf+4HkW3xuvr2dlzmw8i7mt6zlXPR7qWr15tRx3du4wiq/X1Tovr3OOuv4sf44fxc7yZnWNbLV+L3/dR9GKz6LOXr38kXelfrbaS9137/xI/dnYSv6ZvfWL6CG0zrNWj3VMNoqPuDwnxuu4rM6RjfKP4kKec1TUD/V5UffRqt+KC6P4iIs5vdzZmfHFbHxFr/5Kf8WRHkaxUT/0znOOLeNFHitG8fVYS51/RdRpGfVX1LGt86xV50j9Iub0cvRE3VZ8XTfEnNl4Uecv8nhR56nHi5jTGguzOaPx3li5nq/V52FWuyViWuqaWa9Gq4dev9lZc1oirtVbiLHQm9Or34uvr4dRD3lsJb7uqz4v6jyt+gAAAAAAe5z6m05WxQ9Cy9H6Aejoh6A5djZ3pBcf/eSx0Q9p6/hiNj5af7FlvBxbzGLq3KO5LdFvjs1raOWrr+Ue4jzM8hc5vh7LcT15vJ57pH6OiddxnvXizzLLv2W8HGc7Uj/m57E6xyz/zNH6I4+IL+ej8bhe5NfF1vx5LMfE6zh/lFn/5XWc59chznvxRTmP8bPjQ4xHjrCSf2ZWf8t4ObaoY7fGFzlHnGej8Xi9Gp/Hcky8jvOzrPQ3ckZ8xOYcZ8o1cm+5XryO82I2HnL+OA9nr28W3xrvrb8W82qr8T05Z50/545jZDZ+h9H9ifO8tjynvI7z/DqU84jNOYp83hoPOUfOn+fH6zhfNeoPAAAAAOCoWzad3PmDzvih60ger+eOxorZeNG7HkbjK/3fbXYPyrWyjt5aVuLDbHyrWe7iyvrF0fiZWf7R+CM+f0fr5/HW3CP9n1G/KHl6VuJHZvGj8VlsMZvTi3uUcm/jmPVXj5fzuJZfZ/nabLzlSPxZn7+RLflrK/1dbbb+M8fvMOuvKO9Dz0r8USXv3txX9RTOWP9sfaP7X6y+Pz2jr7O9aypy7JE8s/Xfabau0T0o53Etvz7TFTkBAAAAAB7hlk0nR5QfyJYfaMfxip65/zPub47fm2NkS/4rfoC/pf4eV+Yurs5/t7y+2Rrv+HzkmmW87mEWf7dn76/cz9H7erT/LfGtPrbE73E0/5HYo8r9urP+VqPP2VXy/Wndo9xTGa97nMWPvNr7s9WZ65t9Nnrjs/rl+ij3LP6ILfdntv5n0Ooxr2+2xtqW+9NzNH7kjP4AAAAAAHpebtNJUX5wGser/eC09Jv7f0a5vz33N8fHkeV7cEX+q11Z/+rPxyt8/o7K67tjnUfrH42/2rP3l7X+fTna/93xM0fyP8O/D7n+nv8+vLt8f+LY4sz4d3x/7lzfytffqK+V+KNy/nd//+PYIsdtvT/ePwAAAADglb3kppOjZj9ozeNX/lB2b+5H9bfXqKfW2Gx+/cP3u9f8qPpX19mbP8cd6fGq+kd6WnF1/TPjZ7nK+NaHW0f7W1FqHK3TW9fRvM8efzR/2Jsnx53VS8tK7jJn9PlujT+q/72O9vSINZUaz3jvaqXHrf/+FaP1zXKu1uzlL0r8aLwYjZexWfxeJW9vfa2+r+pjr2fp52gfz3ZfAQAAAID39/Hx7fuPz9enKj/wbP3gub7eOm+JOfV4q8bMLEdvPPfaej0bD+U8q8dG50WOL2OtOT117RDx9fhq3qxVI/qM11m+vlK/l78oY62Y0IotonYrT52zlWM0N5+HnKMer/PPahetGj2z/KPzIseXsdackYhvxc7Oi4gvejlqMd7Kt9WR+lmvl1F8a6wY1d8yluu0Xhd1jmI0N5+HnGM03hobadUvWteyuk6dJxvFj+LClvhWvjp+Nl7MespG+bf2V8Zac3pGtUPMaeWexW/JH+r8rZhazrEyP9S1Q91DrbeO3vVazJvVr8db+YuY1xvvqXuuz0PuY8t4vl7UsbPxEPN6tXtxxcp4lufWsa1co/gQc3q5Rq+zXu7W9SznqefWNYpe/WJWK9vaW2j12MpTx/Zq5Xmt3NmsTi++XC+v8/hKbgAAAACAvS7bdAJ71D8kB87j64t3Nvt8+/x/bVe//1/98+frCwAAAADg6/qSf14HAADOMttw8e4bMmw4AQAAAAD4umw6AQAAXlL5LSujAwAAAACAa/nzOgAAAAAAAAAAbGbTCQCX++W3v/t8BQAAAAAAPMrPP/3m8xXANfx5HQAAAAAAAAAANrPpBIBT+a0mAAAAAADwfPz8HrjCl9x0Uv5BfaZ/VM/u5dnWx71e7bOw+vmNefXc1Xi2691vAAAAAACA7IrnB/GcwrMJXs3Zn9utuaL+mT1kHx/fvv/4fH2q3PCj/1ZYqT2qGb09y98wm/W71bOtr3b2ere6u/6jPdt6Z/2sfH5HOVbij3i2+9nT6zPuT6jn9MZzvng9qjHLCwAAAAAAXC//vL73c/2Z+mf8sxx1na3xPaP+ZzWO9iD/MaP8q7XzvFF8cfb6ZvGr4726ZXy1py1za0diRy75TSfRbBxxE59F9PWu3n19vLejn1+f/3/4D1fP7B6t3MNn+3cdAAAAAAC4Rjw3iGPrM4Kj8asif89sfOas/L31n5W/5+r8PWW9kTuOfA9GY6HOUctj9fis/qqIrUWuPF7n78W+i9N/00m8aT0xnm9+rfUm1FrxdVzozenlLddb+UOrTmveSJ2j12NR544eQz7Psb2eVvJnvTw9vfj6esj5y5xyHnNbvbXmZ3WdGK+vh1G+3nnkymOhrtOaM9PLv1K/ONLDLHY0Puqvjgu9OXXdLOrUVuLrPkZ5er1ldfws/976q0psiYv/zWbXVsaLct6aG3p5AAAAAACAx8o/rx/9bH+L2TOCYlRnbx+9uNb1fG02Hsq1YnXu1vzF6tw9+cu14qr8Zxrlr8dmvezpdUtMa26+tjIeWvNa6nkh5s/yjOJ7Y0dc8ptOZmIx5ahvQL5RseDWnFZ8jonXcR5a12q9/EUeizzxv6taOULUy2N1DyOtnC0xr557tH69tshR5PPWeMg5ttTOWvnzeWt81ai/OM+5t65hlL/YMl6OPXrxUS+P1T30+ssx8TrOQ+taVvJFzvw6zOJDzGvNLTljLOfP8+N1nIeYn8dyjpw7zznTlpy5t5Z6PHqOdQAAAAAAANTufoaw9fnHVnvzrz5fedb+71b6iuNOW+vH87WWPBave3NDPV76ybGz+CvcsulkttA83pp79Y26Mn+86SOz9Rd7v5jOqn+lIzVX1nfU1fdvS/7a0fWf8fk4Un+m5I78+fWqrevbI8fvzVXijvYxUu7D6F7MxgEAAAAAgK8lnh3Mni/0rMQfNcs/Gy/XR70dyT+KC0fyF+X6qMbR/Gdp1YjavfpxPY5ynuX4mci1VeSfxa70sMdVec9wy6aTrPWmxBsWR2009g7y+lprzPesjO/5ohiZ1R8pveyNbTl7bWfb8/mdORL7CLm/WY/P/v61rK6tJ8fXOc7++tij1C199N6b1fF4fdc6AAAAAACAx4lnB+UYPRvoPV9Yjd8rnl/06s/GZ+Q/R6nTErXL0Zoz6ivH9uLDaGxkdH+iZhytOWdZqX+H2zedtJQbUx8h3qj6+iPlN+2KHvL64niko/Vz3F0f7Dvl9cex6hk+3zO5v2fuc48z7n+Ob+XJ1+/4+qj7qY3Gc7/5Xt2xDgAAAAAAgHDk+ceKo/lnz1Oevf8zxPofUavlSP2V+xfHHXL9O57bnb7ppLWQrQtbnX/HDSvym7ZH7ru1hqvXNct/133dqtfnWf3vzfNM9ffkmMWctb67rPa/d51n3J+S44r7XP/7XF7nf8dm48Xef/cAAAAAAID31Xqm8Ciz5xuz8VCu53nhrPz1vHBW/nI9zwtX518Rca28K+r+enpjK/XLnFb86v0Js/E9rsh5po+Pb99/fL4+Vb7x+Qas3JAcG+ocWSvfqH5LzKn7a53XWvVH6t721qjjilZsMcq/kqNVv2WWO+R5dW+zWhFb5rXmr/TQq1+M8rfq1er6xSwmq+NH9Vv95Pgy1pozMqpf1ONFzFmtVfcYWrmLVg+tOivx9ZxW7mxWZxZfxJyV3EXM642PjOoXsx564+V6b27r+iwvAAAAAABwvfzz+tbP71fUP+Nv5RjlXolfcaTG6hqK0VjYkz+UuVvjV/LHnNFYODv/SJ07RJ4ttUOeM4uf1Q8xb2v9Io8frV+0xlp9FbPeslGeIy7bdPKuWm/EVW8OwCvq/TsJAAAAAAA8Vv55/VXPNB/1rNQz2ft5Dx7vzHt+1ft3+p/XAQAAAAAAAOBrsAnh/ZXNCjac3OMV7rlNJxuVNzW+qHxxAQAAAAAAADxGPJ/lscrzcM/EX9fVXzf+vA4Al/MNIAAAAAAAPJ6NAsDVbDoBAAAAAAAAAGAzf14HAAAAAAAAAIDNbDoBAAAAAAAAAGCzl9x08stvf/f7AwAAAAAAAACAe5y+6SQ2hNgYAgAAAAAAAADwvj4+vn3/8fn6FGWjyc8//ebz7E/PAQAAAAAAAAB4fZdvOilaG1GylflxnmPruLCSP9s6DgAAAAAAAADw1Z3+53VmYkNH2cgRmznqTR4jOW4k5tVzZ/XL6xwbcwAAAAAAAAAA+AcP33RS5I0cvU0dWzaiZLFpZGSlPgAAAAAAAAAAfbdsOikbQ/JRyxtBVjaRbDWqX2r1xgAAAAAAAAAA+IPbftNJfTzSrH6+buMJAAAAAAAAAMCfunzTSdm0UW/quHojxyy/jSQAAAAAAAAAAMd8fHz7/uPz9SnqDR31hpOitemjN291w0qeN+thVH8WCwAAAAAAAADABZtOAAAAAAAAAAB4f5f/eR0AAAAAAAAAAN6PTScAAAAAAAAAAGxm0wkAAAAAAAAAAJtduunkl9/+7vMVAAAAAAAAAADvxG86AQAAAAAAAABgM5tOAAAAAAAAAADYzKYTAAAAAAAAAAA2s+nkBr/89nefr445Kw8AAAAAAAAAwFY2nbwoG04AAAAAAAAAgDu93aaT2IxR/jeOlt5YXFuJbc2J895YPV7PWVFifv7pN59nAAAAAAAAAACP95a/6SQ2ZcTGjHpjRx5vbfoYjcd5jBetOa34HBOv43yLPTEAAAAAAAAAAGf6+Pj2/cfn69PVmzGOWtlsERs+sta1UI9tPS/yta3zjzgrDwAAAAAAAADAVpdvOnn0poiVTR7lPKvHRj3XsSFiWrXqfLMaq87KAwAAAAAAAACw1Vv+eZ2R2KgRxx45/kgeAAAAAAAAAIBX9fabTka/DaSM7bE3DgAAAAAAAADgXbzln9fJWvVHc1Z6bm06iZg6vpcv55jVq43qAwAAAAAAAABc4Z/8+//6+epXv/o//+HfvuemExswAAAAAAAAAADOVW86efs/rwMAAAAAAAAAwPlsOnki5be0jA4AAAAAAAAAgGfxdn9eBwAAAAAAAACA832pP6/jt4MAAAAAAAAAAFzDn9cBAAAAAAAAAGCzt/7zOv68DwAAAAAAAADANfymkwdb+ZM/ZU5vXoyt5Alb5gIAAAAAAAAArLDp5MWU39zyqN/eYrMKAAAAAAAAANBj08kTeuTGkh4bTgAAAAAAAACAEZtOLhB//iaOlt7YLG7VkTwl5u5NLwAAAAAAAADAc7Pp5CLx20pamzdiU0c56k0hvZgtcv49uY7WBwAAAAAAAADe38fHt+8/Pl+f7u7fmPGf/vN/+dVP//SffZ4dt7KW2Zrr8d78rXlC6/osV8/eOAAAAAAAAADg/b31ppM7zNZcj/fmb80TWtdnuXr2xgEAAAAAAAAA78+f1wEAAAAAAAAAYDObTi5QfkPInXL9u3sBAAAAAAAAAN6TP69zgXqjR74H9T1pnbfEnNl4keeU63WNmVaNLfEAAAAAAAAAwPuz6QQAAAAAAAAAgM38eR0AAAAAAAAAADbzm06+kNafzcm8VwAAAAAAAADAKptOAAAAAAAAAADYzJ/XAQAAAAAAAABgM5tOAAAAAAAAAADYzKYTAAAAAAAAAAA2s+kEAAAAAAAAAIDNbDq5wS+//d3nq31KfD4AAAAAAAAAAB7NppMXUzaZ/PzTb/7osPEEAAAAAAAAAHi0t9t0Ehsw4reA9DZk9Mbi2kpsa06c98bq8XrOTNlkAgAAAAAAAABwt7f8TSdlI0f8FpA4z/J4a9PHaDzOY7xozWnF55h4HecAAAAAAAAAAK/k4+Pb9x+fr0/3n/7zf/nVT//0n32eHbeyQSM2fGSta6Ee23pe5Gtb5x91Zi4AAAAAAAAAgFWXbjq5w8omj3Ke1WOjTRx1bIiYVq0636zGqujljFwAAAAAAAAAAFu85Z/XGYkNH3HskeOP5DnChhMAAAAAAAAA4E5vv+lk9FtFYuPGVnvjzmLDCQAAAAAAAABwt7f88zpZa2PGaE4Zm23maG06iZg6vpcv55jVy1q1iy05AAAAAAAAAACOestNJzZgAAAAAAAAAABc6+3/vA4AAAAAAAAAAOfzm06eSO9P5wS/wQUAAAAAAAAAeBZvt+kEAAAAAAAAAIDr+fM6AAAAAAAAAABsZtMJAAAAAAAAAACb2XQCAAAAAAAAAMBmNp0AAAAAAAAAALCZTScAAAAAAAAAAGxm0wkAAAAAAAAAAJvZdAIAAAAAAAAAwGY2nQAAAAAAAAAAsJlNJwAAAAAAAAAAbGbTCQAAAAAAAAAAm9l0AgAAAAAAAADAZjadAAAAAAAAAACwmU0nAAAAAAAAAABsZtMJAAAAAAAAAACb2XQCAAAAAAAAAMBmNp0AAAAAAAAAALCZTScAAAAAAAAAAGz28fHt+4/P1wC8qV9++7vPVwAAAAAAAOf4+afffL4Cviq/6QQAAAAAAAAAgM1sOgF4M36rCQAAAAAA8GieT8DX9CU3nZR/8J7pH72ze7l7fVvrP7rXo/fnaPyzeda1xH2u+2td+yp69wMAAAAAAOArueL5SDyH8eyFs539udqaK+qf2UN22aaTqxsfuaMmz+Po++/zc7/yHpS/ARjHK+p9jsr1fNR64+V13I+4HtcAAAAAAABeVTwTiWOmfj6yNX4knsXUZjVm4zOz+Nn4zCx+Nj4zi5+NP8rR2q34vK69+bfElblbnw/2PtdnuWTTSSw0jr039ypX39S73b2+Z7+/R/t79vW9u1e4/7N/82ZrWFnjs/27CgAAAAAAsEc8F4lj6zOQo/GrIn/PbHymF1/WE2NxjNbYG4vYntn4zNX5jzr6uZjd1zha82LsXX18fPv+4/P1KcpNHN2wGI+b3ZpbvxGjOXmsjgu9Ob285Xorf2jVac0bqXP0eizq3NFjyOc5ttfTSv6sl6dlpX4xqjEbK+cxZxQXenPy9eJofKjzzOr08oxEjlbucq03Ho70MIsdjW/przWWc49ia3UP2ahOr8ZI9B7/m82urYwX5bw1N/TyAAAAAAAAnCk/jxg9u9hi9gykGNXZ20cvrnU9X5uNh3KtWJ1bX8t646u58rUtMcVV+a8U9fbW3RJfzynnoY7NY1mvRsyf5RnF98aOuOzP64zEYspR34B8o2LBrTmt+BwTr+M8tK7VevmLPBZ54n9XtXKEqJfH6h5GWjlbYl499xH1j64/x+exHBOv4zy0roU8Fq/rua1r2az/3Hues0Vv/WHLeDn26MVHvTxW9zDqr5zHtfw65LwtZX7MiXl5fj2ex86yJWe9vlpv/bEOAAAAAACAr+buZyRbn+9stTd+9fnR3f0fzb/H0c/MkfgS24vPY/G6NzfU4/G+r8Zf4ZZNJ7OF5vHW3Ktv1JX5400fma2/2PvFeFb9vc6of3ZPZ5v1f9Qs52h85f6PXP3+lbEYz68f7era5T6O7uVsHAAAAAAA4J3Es5HZ85OelfijZvln4+X6qLdZfGjNWY0bzZuNl+ujGkfzv7JY25XrK7mf1S2bTrLWTc9vSuvmjcbeQV5fa435npXxsz+4s/pXu7v+UaP+y3vVG1t1NP4rO+P+H1Xqlj56X7er4/H6rnUAAAAAAACcJZ6NlGP07KP3/GQ1fq94PtOrPxufWY3fu7a7+z+a/9nF2sqx9z1a0bp/UTOOO9y+6aQlvylxhPyBzNcfKb9pV/SQ1xfHI331+kfN+s/Xt37hP8Pn/9nNvj7z/dt6/88we99G47nfWN9d6wAAAAAAAPgqjjzfWbESH8+DWnNnz4vu7v9ofsbK/Y3jjueGp286aS1k68JW599xw4r8pu2R+26t4ep1zfI/sv4d67/ao/rfW2d2/2dmMUfzH3X067MofV/Re+mpvj+5z9l4cWRdAAAAAAAAr6j1zORRZs9vZuOhXM/zwkp8jLfyhjpPmOWfjYdyPc8LV+d/lLvrj/Tu2bP4+Pj2/cfn61PlNyTfgJUb0noz6xxZ70MZRrEh5tT9tc5rrfojdW97a9RxRSu2GOVfydGq37K1frlezkf9FTHempvPQ10jtHIXdY4j8a05MV6P1XVXjHKUsdF5kePLWGvOyKh+MRqva/Vqj663xNzWeF0va9UoYl5vfORoD73xcr03t3V9lhcAAAAAAOCo/Dyi9XxiRf0Mo5VjlHslfsWRGqtrKEZjIc+px8Ko1639zcaLmDMaC2fnP6quX/R6XFlfEfNma2vFFqN5rbFWX8WsfjbKc8Rlm07eVeuNuOrNAbbx9fkHvfsAAAAAAABwpvw84qpnMo961vMVnyk9G+/Bnzrznlx1f0//8zoAAAAAAAAAcAabEN5f2Qxhw0nbK9wTv+lkh/KBz3z44Xn4+mzvUqzvCwAAAAAAwFH5ecSrbxrIz1I8/+WdXP3ZtukE4Auw6QQAAAAAADibzRmATScAAAAAAAAAAGz268//BQAAAAAAAACAZTadAAAAAAAAAACwmU0nXO7v/vqvPl+9htJvr+cYO3NNZ+Y60xVrnbmi1tk5H31Pvqq4z890r2c9zcb3ODNXcXZ/AAAAAAAAfG2XbTrxYIs7xOcuH2f69V/85e+PO529ppZSI9a6db2P6I/3duTzd6VZP8/WLwAAAAAAAFztkk0n8cAQ7hAPfvc+AN4bx9fg88Er8/kFAAAAAADgTB8f377/+Hx9mth00tp8kseKPQ+/IjbkHLP8K/WP5F8xyl8cGY/+Qu88crRyj8aLGAt751wh+p/prS/3PcozqrOy9pU5tTomjNZQrOSu7VnfGf1F3ZjT62FmViNr9dfqKV6HXm8r+bNenp6j+UfjZayc5zln5l8VfbSM8kdczNlTu5jFj/orZuMzUT/kXKOxoq6dz3Nsr7+V/NnWcQAAAAAAAN7L6ZtO4oFTedCUX4f6Wjnf8lCqFV+MznP+PePF6DzHz8zyFbOco/F6rHVexLUzxot6zuj8Siu18pze/FmeUVyR8xd5bh07q1UbzV+pPxLzs7rXYpS/XBvVG4238o9ytdQx+byVv+jNL+rzonUtzMaKGK/PZ0a5w5H6Z5zH6z0iX5bzPaKfHNOLn+XdUzfUsXU/Re+8GMWH1rUwGytivHXeiwUAAAAAAOA9XfLndeKhU+/h09GHUjm+lWuWf8v4nvwzo/yzh3ZnPNSbxW8Zr+c+w0PH0kM+alf3d/f9GdWfKfMjJr/O8rXW+MjK+rfmzLbm781tfW5WnFV/rzPqHx0/ouSLnPl1lq/Nxvc4Gn/EV//8AgAAAAAA8Hou2XRytfJgLB8jWx6g7fHqD9323J8t93+Po/lLz/moHcm9Iue/qsadXn19s/7zZ6aMtz5DR8zqj5Re9saeJdeve3iG/rI9790z9d+S+2v1eOfn99nefwAAAAAAAK536qaTeMhUP3Q6++FTebBVHzzO1ff/ytzls3hV7pDzX1nnLq++vrv7P1o/x539b+uKXD+OLF+/o78jHvHvw1G5vzv6nNXP11/t/QcAAAAAAGC703/TSX7gFMfZrn6QlfNfUWuWf1Zztacrei+29HdVD2d41/vzyPx7XNFTNlv/I+u3PFP9lbn1v+FX9z+zpf8j7lrnbH1X9zXLf9d9AQAAAAAA4Dl9fHz7/uPz9WHlYVRrk0m+Xs/pxYy0Hnr18tdW6tX5j/ZbG+Uvzhov1+t+6/PabLyo6xd1jdDq4Up7+u/1nsWc2XjRmtMbL9fLeR5fUefI6vpbcxejnur8xaiHLf3VdUd99NS1WzlrrRqt2q3YYpR/JUerfsssd7G1/pbei1H/K/ErSp5e7Kz/vTXDLH9LzJmNr8g5Sly9plaNVv46rljpr56zkiPmzGIBAAAAAAB4P6duOgHgdbU2KgAAAAAAAAD0nP7ndQAAAAAAAAAAeH82nQAAAAAAAAAAsJk/rwMAAAAAAAAAwGZ+0wkAAAAAAAAAAJvZdAIAAAAAAAAAwGY2nQAAAAAAAAAAsNnlm05++e3vPl89h7P6+ef/7V/9/bHVKG4l72z81fy7//n/Pl9dr9R6ZL1n8I7rfdb3Mfqa9Tcbf3fPsv6f/+ZvP1+9rr/767/6/XGVq/PPXFH/rjXdUTfXjNf52hZnfr082/eGofR1V29H617R99k5776/s/p39vcMRuuPsdH9mY0fcTTvVX3xPEafvxgbfQ5m4wAAAACMTTed/N9//S/+6Nii/ODm559+83n2HEo/R3+gVDZ8/O9/8z/+/jhzA0jkPGK1n7M3rux5kPWVH7w/gvv7p67c8PAf/+U/+v3xzK5c/yspD9B/+fM/+zx7nL0P/OGoX//FX/7+2Kt8vRzdeJIf6r3jw713XNO7KN//X/1/k7zz+3/G/Xv1+3N1/2fl7+Up1/NRm42/Mp9fAAAAgOsNN52UTSb/+L//rz86VjeelB/MXP3D3b1KX3t/cBQbTrKtG0+Obiw5Gl+cveEk2/pA9dkf0r+6d7y/r7C5Y+TV+z/q7vXfteGE53B088VXdmTjSXxfGN8bPuP3iLk/zvfs9/erv/9H1//V798zmP3ft7P36JXfw6O9v/LaAQAAAJ7Bx8e37z8+X/+R2HAS/9u71jPadFKPtc6z1tyY06oxy1+0rq1obTop8vV6Tj0WRhtHRnVCPZ7Hsta8cq1X44i84WT2UC9+20I8fC7nrQfRrev1b2rIOYpyXucv6mujOWE0FvKcIs+rx4pRjWJLD638Rcyb1R/lDjGnzjk6r8eyuk5rTstqXGtefS2s1s5mfcR473rYU7sYrW9Ws9bqoddnK9esXh4fjWVH6q/obTppbZhr/TtWz8tzemO9zXhb8xerffYcyb+yjln+mdn6evnjel2vXC/XVvuazVvNU8v95RwRn8eL3nnWqj2bV+et1fH1vL2btvL3b3u+/wolvmh9X1fbUifH9+LqGjGvVbvYWr/Mj1x1bIyH3nnEF3WOFTm+qGtkdf5eT/E69PpayZ/18ozUPYYj/dXXw57+eqLvqLUnd+QI+TyvYZS7zhFG8XksG9WpRd3I1aqRr/XOI76oc/TkmKzVQ7Y6vpp/puQpMfG/2ezaaswRJV9tS/66n7r/MMrZW9MoPo9lozoAAAAAX8+vfvX/AXgWYyeWrwo9AAAAAElFTkSuQmCC">